lua_examples/luaOTA/luaOTAserver.lua

```diff
@@ -162,6 +162,10 @@ end
 ----------------------------------------------------------------------
 receive_and_parse = function(esp)
   local line = esp:receive("*l")
+  if (not line) then
+    error( "Empty response from ESP, possible cause : file signature failure, check that the secret on ESP and server match", 0)  
```

Comment:
Our travis test build fails with the following messages:

> Checking lua_examples/luaOTA/luaOTAserver.lua     2 warnings
>     lua_examples/luaOTA/luaOTAserver.lua:166:121: line is too long (130 > 120)
>     lua_examples/luaOTA/luaOTAserver.lua:166:129: line contains trailing whitespace

To fix them just push your commits to your branch. They will show up here.